ui/app/peers/create/page.tsx

```diff
@@ -1,21 +1,33 @@
+'use client';
 import { Action } from '@/lib/Action';
 import { Button } from '@/lib/Button';
 import { ButtonGroup } from '@/lib/ButtonGroup';
 import { Icon } from '@/lib/Icon';
 import { Label } from '@/lib/Label';
 import { LayoutMain, RowWithSelect } from '@/lib/Layout';
 import { Panel } from '@/lib/Panel';
-import { Select } from '@/lib/Select';
+import Link from 'next/link';
+import { useRouter } from 'next/navigation';
+import { useState } from 'react';
+import SelectSource from '../../../components/SelectSource';
 
 export default function CreatePeer() {
+  const [peerType, setPeerType] = useState<string>('');
+  const router = useRouter();
   return (
     <LayoutMain alignSelf='center' justifySelf='center' width='xxLarge'>
       <Panel>
         <Label variant='title3'>Select source</Label>
         <Label colorName='lowContrast'>
           Start by selecting the data source for your new peer.
         </Label>
-        <Action icon={<Icon name='help' />}>Learn about peers</Action>
+        <Action
+          icon={<Icon name='help' />}
+          href='https://docs.peerdb.io/sql/commands/create-peer'
+          target='_blank'
+        >
+          Learn about peers
+        </Action>
       </Panel>
       <Panel>
         <Label colorName='lowContrast' variant='subheadline'>
```

Comment:
I think both source and Datasource is not required any one is sufficient
![image](https://github.com/PeerDB-io/peerdb/assets/39798643/c3929ffb-8d08-4d0b-ba29-53c4243a2a74)

Reply:
Yup. Changed. Thanks